### PR TITLE
less simplifications

### PR DESCRIPTION
**Pull request by fredj**
Small simplifications in the less files

### CODE REVIEW

**Comment by ger-benjamin** [comment]
Do you plan to fix https://github.com/camptocamp/ngeo/issues/1277 by the way ?

**Comment by fredj** [comment]
Yes, I will also fix #1277

File: contribs/gmf/less/desktop.less [hunk 1]
```diff
@@ -224,18 +224,18 @@ main {
 
 
 ::-webkit-scrollbar-track {
-    -webkit-border-radius: @app-margin;
-    border-radius: @app-margin;
-    background: @main-bg-color;
+  -webkit-border-radius: @app-margin;
```
Comment (pgiraud) [inline contribs/gmf/less/desktop.less]: Won't the auto prefix take care of that?

File: contribs/gmf/less/layertree.less [hunk 2]
```diff
@@ -34,10 +36,10 @@
       color : @color-light;
       display: inline-block;
       &::before {
-        content: "\f054";
+        content: @fa-var-chevron-right;
```
Comment (pgiraud) [inline contribs/gmf/less/layertree.less]: Awesome. I didn't know there were variable for the unicode codes in font-awesome. Nice catch.

File: contribs/gmf/less/desktop.less (hunk 1 above)
Comment (fredj) [inline contribs/gmf/less/desktop.less]: you're right, I will change that

**Comment by fredj** [comment]
Thanks for the review

**Pull request merged**